docs/content/documentation/getting-started/cli-usage.md

```diff
@@ -72,6 +72,10 @@ the interface/port combination to use if you want something different than the d
 
 You can also specify different addresses for the interface and base_url using `--interface` and `-u`/`--base-url`, respectively, if for example you are running Zola in a Docker container.
 
+> By default, devices from the local network **won't** be able to access the served pages. This may be of importance when you want to test page interaction and layout on your mobile device or tablet. If you set the interface to `0.0.0.0` however, devices from your local network will be able to access the served pages by requesting the local ip-address of the machine serving the pages and port used.
+>
+> In order to have everything work correctly, you might also have to alter the `base-url` flag to your local ip.
```

Comment:
Sorry for barging in, but I'd like to suggest making this suggestion more forceful, because you definitely _have_ to use the `--base-url` option to make it work, otherwise even though the HTML is served from the IP specified with `--interface`, it uses `127.0.0.1` to reference to CSS, JS etc, which is not going to work (as I've just discovered).

Alternatively I wonder if `zola serve` could be smarter and use the correct interface automatically if the base URL is a local IP?

Reply:
It would be nice to avoid having to specify multiple parameters, i'd take a PR for that